parent/pom.xml

```diff
@@ -373,7 +374,7 @@ Bundle-Category: jackrabbit
             <dependency>
                 <groupId>org.apache.jackrabbit</groupId>
                 <artifactId>oak-jackrabbit-api</artifactId>
-                <version>${oak.version}</version>
+                <version>${oak.version.compile}</version>
```

Comment:
Actually we state that we are backwards compatible with Jackrabbit 2.16+ (in https://jackrabbit.apache.org/jcr/downloads.html#vlt) so if we distinguish let us rather only manage that dependency in that version and not depend on the newer Oak one.

Reply:
Jackrabbit 2.16 has reached End Of Life.

Reply:
Not convinced. Why would we reference an unmaintained version?

Reply:
I know that you disagree here, but again, that is the job of the distribution to ship current runtime dependencies. Applications only depend on the minimum API level version which is required (as long as the newest version is backwards compatible with it, i.e. fulfills the package-import version range). No need to for FileVault to no longer support being compatible with unmaintained API versions.

Reply:
I am only talking about the OSGi bundles, for VLT we should ship the newest dependencies.

Reply:
> I know that you disagree here, but again, that is the job of the distribution to ship current runtime dependencies.

Yes. But. Shouldn't we encourage users of FileVault to sop using unmaintained versions?

Reply:
By stop supporting running on older versions? No! By writing that somewhere and referring to the maintained versions? Yes

Reply:
Well, I continue to disagree. The audience we're talking about is not reading documentation until something breaks.

Reply:
Funny thing is: The runtime version range for used packages of Jackrabbit API is the same for 2.16 and the newest one. So even depending on the newest doesn’t make any difference (no one will notice). Also Oak 1.48 is unmaintained as well!

Reply:
Also we deliberately support Java 8 although it is  unsupported these days. Not even Oak does encourage adoption of Java 17 by no longer supporting Java 11 ;-). For me that falls in the same category.